test/pylib/scylla_cluster.py

```diff
@@ -291,19 +292,17 @@ async def cql_is_up(self) -> bool:
             caslog.setLevel(oldlevel)
         # Any other exception may indicate a problem, and is passed to the caller.
 
-    async def rest_api_is_up(self) -> bool:
+    async def get_host_id(self, api: ScyllaRESTAPIClient) -> bool:
         """Test that the Scylla REST API is serving. Can be used as a
```

Comment:
Docstring needs updating.

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/0ebda403004c746f8a0019098e9125c160433847 https://github.com/scylladb/scylladb/compare/6635e6163463bf74df6826ee15735d79e6493578..0ebda403004c746f8a0019098e9125c160433847